### PR TITLE
RSWEB-8025: refactor features pattern

### PR DESCRIPTION
**Pull request by writes**
Updates feature pattern to better use common css class naming convention.

- removes nested classes from wrapper class
- sets underline for links on the features pattern to the `.feature-bodyWrapper` class so that links are not dependent on the global `p > a` style assignment

http://localhost:9000/derek/solutions/solutions

### CODE REVIEW

File: styleguide/_themes/global/scss/fonts.scss [hunk 1]
```diff
@@ -2,11 +2,13 @@ $heading: 'Khand', Helvetica, Arial, sans-serif;
 $subhead: 'Khand', Helvetica, Arial, sans-serif;
 $copy: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
 $code: Tahoma, Arial, Geneva, Helvetica;
+$font-rsweb: 'rsweb';
```
Comment (brantdarilek) [inline styleguide/_themes/global/scss/fonts.scss]: @writes 
Why add in the variable here? It looks like you are declaring it here but not using it.

Comment (writes) [inline styleguide/_themes/global/scss/fonts.scss]: @brantdarilek Good call. I was originally going to attempt to remove the need for the `.rsweb` overwrite but the specificity was causing problems. I meant to remove but didn't. I have updated the PR. Let me know if you see any other edits. Thanks!

**Comment by WidgetsBurritos** [comment]
👍

**Comment by runrog** [comment]
@brantdarilek @WidgetsBurritos My styling changes have been merged with this branch (sass file). Can you give a quick review

**Comment by WidgetsBurritos** [comment]
👍 on @runrog's updates.

**Comment by amberRrucker** [comment]
Looks good. Can we add the caveat that ECMs are going to need to go check what BG color these are on because I know there are some on the dark gray

**Comment by amberRrucker** [comment]
+1

**Pull request merged**